### ISSUE TITLE
Database error when restoring previously-deleted nav

### ISSUE THREAD

**Issue by elivz** [verbb/navigation#365]
### Describe the bug

When a project config application restores a previously-deleted navigation, there is an incorrect table name in a query that throws and error and does _not_ restore the previous nodes. 

On [line 384 of services/Navs.php](https://github.com/verbb/navigation/blob/f5223c13f9fb8007188c6e1b66c01cd81b540747/src/services/Navs.php#L384), I believe

```
->andWhere(['nodes.deletedWithNav' => true])
```

should instead be:

```
->andWhere(['navigation_nodes.deletedWithNav' => true])
```

That `nodes` table alias is not present in the query, so the WHERE condition fails currently.

### Steps to reproduce

* Create a navigation & commit the project config changes that are generated
* Switch to a different branch and run `./craft project-config/apply`, which will *remove* that navigation as the config files are no longer present
* Switch back to the original branch and again run `./craft project-config/apply`. The plugin will attempt to recreate the deleted nav, but will fail due to a DB query error: `Column not found: 1054 Unknown column 'nodes.deletedWithNav' in 'where clause'`

### Craft CMS version

4.5.0

### Plugin version

2.0.20

### Multi-site?

Yes

**Comment by engram-design** [comment]
Absolutely correct! Fixed for the next release. To get this early, run `composer require verbb/navigation:"dev-craft-4 as 2.0.20"`.

**Comment by engram-design** [comment]
Fixed in [2.0.21](https://github.com/verbb/navigation/releases/tag/2.0.21)
